src/network/networkd-address.h

```diff
@@ -62,7 +63,7 @@ struct Address {
         LIST_FIELDS(Address, addresses);
 };
 
-int address_new_static(Network *network, unsigned section, Address **ret);
+int address_new_static(Network *network, const char *file_name, unsigned section, Address **ret);
```

Comment:
It's a minor thing, but here you use `file_name`, and `filename` everywhere else. We consistently use `filename` in sources.